vuepress/docs/next/docs/reference/local-tips-and-tricks.md

```diff
@@ -48,7 +52,7 @@ The namespace will be recreated, preserving the images already pulled, so it's u
 ### Entando in Kubernetes
 **Q: How can I install a new copy of Entando into an existing VM?** 
 
-**A:** By default, the quickstart installation deploys Kubernetes resources into a dedicated namespace called `entando`. If you want to remove all of the resources in `entando`, you can simply delete the namespace with `sudo kubectl delete namespace entando`. You can then re-create the namespace and re-install the resources. Alternatively, you can achieve this with `ent quickstart --vm-reuse=true`, but you'll need to set other `ent quickstart` options, so check the `ent` help.
+**A:** By default, the quickstart installation deploys Kubernetes resources into a dedicated namespace called `entando`. If you want to remove all of the resources in `entando`, you can simply delete the namespace with `sudo kubectl delete namespace entando`. You can then recreate the namespace and reinstall the resources. Alternatively, you can achieve this with `ent quickstart --vm-reuse=true`, but you'll need to set other `ent quickstart` options, so check the `ent` help.
```

Comment:
I would switch sudo kubectl to ent kubctl since ent quickstart is used in the same paragraph

Reply:
@nshaw many other paragraphs on this page use `sudo kubectl`... not a problem / worth changing? contexts aren't specified for the commands

Reply:
@Lyd1aCla1r3 just seemed odd in the same paragraph but probably not worth adjusting the whole page.